Dockerfile

```diff
@@ -13,10 +13,11 @@ COPY client ./client
 COPY config ./config
 COPY public ./public
 COPY preact.config.js .
-RUN yarn && yarn build && rm -r node_modules
```

Comment:
This was intentional, shouldn't be changed. `yarn build` requires dev dependencies but to reduce image size, we remove them after building. Then we set environment to production and run `yarn` again to reinstall production dependencies for server.

Reply:
This was just me debugging and trying to figure out what was going on, dw